Assets/HoloToolkit/Input/Scripts/Interactions/TapToPlace.cs

```diff
@@ -181,6 +195,18 @@ private void ToggleSpatialMesh()
             }
         }
 
+        public void CalculateColliderOffset()
```

Comment:
Could move the logic of this method into an extension which would it make also possible to test it. Maybe something like transform.GetColliderBounds.
I have an older project with a similar method where I have some tests for it. I could include them after this gets merged.